backend/db.js

```diff
@@ -184,7 +184,7 @@ async function importUnregisteredFiles() {
     // run through check list and check each file to see if it's missing from the db
     dirs_to_check.forEach(dir_to_check => {
```

Comment:
`dirs_to_check.forEach(dir_to_check => {` needs to be `dirs_to_check.forEach(async dir_to_check => {`

Reply:
I'm going to use a for-of loop here to make sure the code is run serially rather than in parallel. In the future we could switch to using parallel with `Promise.all` but I want to try and avoid potential bugs in this PR :)